src/web/controllers/admin/event_admin_controller.py

```diff
@@ -66,7 +66,7 @@ def _admin_event_tab_render(
         match modal:
             case None:
                 pass
-            case 'event':
+            case 'event':                
```

Comment:
woops, I think that's a mistake
```suggestion
            case 'event':
```